plugins/trace-activation/instrument.go

```diff
@@ -47,6 +47,17 @@ func (i *Instrument) VersionChecker(version string) bool {
 }
 
 func (i *Instrument) Points() []*instrument.Point {
+	var instPoints []*instrument.Point
+	// append toolkit/trace related enhancements Point
+	instPoints = append(instPoints, tracePoint()...)
```

Comment:
If you want to instrument trace and logging in the one plugin, please rename the name of the plugin to the `toolkit-activation`.